modules/vcf/templates/annotate.sh

```diff
@@ -224,6 +224,54 @@ vep() {
   ${CMD_VEP} "${args[@]}"
 }
 
+viab(){
+  local -r input="${1}"
+
+  zcat "${input}" | awk 'BEGIN {OFS="\t"} 
```

Comment:
I think the input separator should be defined as well to ensure that this works on all platforms:
`awk -v FS='\t' -v OFS='\t' '<other stuff>'`